docs/csharp/misc/cs0236.md

```diff
@@ -1,36 +1,38 @@
 ---
 description: "Compiler Error CS0236"
 title: "Compiler Error CS0236"
-ms.date: 07/20/2015
-f1_keywords: 
+ms.date: 06/26/2021
+f1_keywords:
   - "CS0236"
-helpviewer_keywords: 
+helpviewer_keywords:
   - "CS0236"
-ms.assetid: 1522c421-744f-441f-9e05-6bb31311ab2a
 ---
 # Compiler Error CS0236
 
-A field initializer cannot reference the nonstatic field, method, or property 'field'  
-  
- Instance fields cannot be used to initialize other instance fields outside a method. If you are trying to initialize a variable outside a method, consider performing the initialization inside the class constructor. For more information, see [Methods](../programming-guide/classes-and-structs/methods.md).  
-  
- The following sample generates CS0236:  
-  
-```csharp  
-// CS0236.cs  
-public class MyClass  
-{  
-   public int i = 5;  
-   public int j = i;  // CS0236  
-   public int k;      // initialize in constructor  
-  
-   MyClass()  
-   {  
-      k = i;  
-   }  
-  
-   public static void Main()  
-   {  
-   }  
-}  
+A field initializer cannot reference the non-static field, method, or property 'name'
```

Comment:
```suggestion
A field initializer cannot reference the non-static field, method, or property 'name'.
```

Reply:
@IEvangelist In all compiler message articles, this first paragraph is an exact match of the error message of the compiler (with the same punctuation). But I'm going to apply the change.